### PR TITLE
Check to ensure user is not disabled before sending the email update notification

### PR DESCRIPTION
**Pull request by brianteeman**
The Joomla plugin that checks for updates and sends emails to SuperUsers informing them that an update is available only checked to see if the superuser has sendEmail set to Yes. It did not check to ensure that the user was not blocked (disabled).

This PR adds a check to ensure that the user is not blocked and so it only sends the email notification to users who are 

1. User Group = SuperUsers
2. User Status = Enabled 
3. Receive System Emails = yes

### CODE REVIEW

File: plugins/system/updatenotification/updatenotification.php [hunk 1]
```diff
@@ -360,6 +360,7 @@ private function getSuperUsers($email = null)
 							)
 						)->from($db->qn('#__users'))
 						->where($db->qn('id') . ' IN(' . implode(',', $userIDs) . ')')
+						->where($db->qn('block') . ' = ' . (int) 0)
```
Comment (andrepereiradasilva) [inline plugins/system/updatenotification/updatenotification.php]: if you put an integer directly you don't need to convert it to int
just is enough
```php
->where($db->qn('block') . ' = 0')
```

Comment (brianteeman) [inline plugins/system/updatenotification/updatenotification.php]: That code basicaly came from 
https://github.com/joomla/joomla-cms/blob/staging/components/com_users/models/registration.php#L620

Comment (brianteeman) [inline plugins/system/updatenotification/updatenotification.php]: Updated thanks

**Comment by wilsonge** [comment]
Merged on review

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**